pyiceberg/io/fsspec.py

```diff
@@ -173,6 +173,7 @@ def _adlfs(properties: Properties) -> AbstractFileSystem:
 
 
 SCHEME_TO_FS = {
+    "": _file,
```

Comment:
pyarrow defaults scheme to `file` when no scheme is present. 
https://github.com/apache/iceberg-python/blob/8464d713d69e6e2f4fa6f629362a27c940a097e6/pyiceberg/io/pyarrow.py#L322

we essentially do the same here